README.md

```diff
@@ -1,36 +0,0 @@
-[![CircleCI](https://circleci.com/gh/projectglow/glow.svg?style=svg&circle-token=7511f70b2c810a18e88b5c537b0410e82db8617d)](https://circleci.com/gh/projectglow/glow)
-
-# Building and Testing
```

Comment:
Ok, I think I was incorrect before when I said that we should get rid of this. This is important information that people need to contribute to the project. I guess the Pypi readme is more about usage.